docs/workflows/best-practices-organization.md

```diff
@@ -0,0 +1,97 @@
+# Guidelines on how to organize for quality software
```

Comment:
Looking at the file name, but you could call this best practices instead of guidelines?